tests/rbd/rbd_utils.py

```diff
@@ -444,9 +444,16 @@ def move_image_trash(self, pool_name, image_name):
             pool_name : name of the pool
             image_name : mane of the image
         Returns:
-
+            0: on success
+            1: on failure
         """
-        return self.exec_cmd(cmd=f"rbd trash mv {pool_name}/{image_name}")
+        cmd = f"rbd trash mv {pool_name}/{image_name}"
+        rc = self.exec_cmd(sudo=True, cmd=cmd)
+        if rc != 0:
```

Comment:
If condition could just be 
if rc:
   log.error()

Reply:
Done.